### PR TITLE
Added github actions

### LINKED ISSUE #2: Add travis or github actions

### PR DESCRIPTION
**Pull request by elyutikov**
Closed #2

### CODE REVIEW

File: .github/workflows/basic.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+name: vicly-basic-pipeline
+
+on:
+  push:
+    branches: [ master, develop ]
```
Comment (Ssstlis) [inline .github/workflows/basic.yaml]: Is this check out our PRs?

**Comment by Ssstlis** [comment]
I've just opened GitHub Actions and i suggest you should add this lines for yaml file:
```yaml
  pull_request:
    branches:
      - *
```

File: .github/workflows/basic.yaml (hunk 1 above)
Comment (elyutikov) [reply .github/workflows/basic.yaml]: No, this is a check only push. Do we need a check out our PR?

Comment (Ssstlis) [reply .github/workflows/basic.yaml]: Of course we are =) I put a suggestion below

**Pull request merged**